fastlane/Fastfile

```diff
@@ -106,6 +106,9 @@ platform :ios do
 
     ios_bump_version_release
     new_version = ios_get_app_version
+    extract_release_notes_for_version(version: new_version,
+      release_notes_file_path: FIle.join(ENV["PROJECT_ROOT_FOLDER"], 'RELEASE-NOTES.txt'),
```

Comment:
```suggestion
      release_notes_file_path: File.join(ENV["PROJECT_ROOT_FOLDER"], 'RELEASE-NOTES.txt'),
```

This wouldn't have happened in _insert compiled language_ 😢 I wonder if there's some kind of static analysis that we could put in place to catch this kind of stuff for us.

Reply:
Took me a while to see what was the difference in your suggestion diff 😅 this `I` vs `i` was so sneaky 😰 good eye!

Wonder if rubocop would have caught this? (We don't yet lint our Fastfile, but et definitively should)